lib/plugins/aws/remove/lib/bucket.js

```diff
@@ -1,13 +1,12 @@
 'use strict';
 
-const BbPromise = require('bluebird');
-const { legacy } = require('@serverless/utils/log');
+const { legacy, log } = require('@serverless/utils/log');
 
 module.exports = {
   async setServerlessDeploymentBucketName() {
-    return this.provider.getServerlessDeploymentBucketName().then((bucketName) => {
-      this.bucketName = bucketName;
-    });
+    const bucketName = await this.provider.getServerlessDeploymentBucketName();
+    this.bucketName = bucketName;
+    this.checkIfBucketExists(bucketName);
```

Comment:
What's the purpose of that call?

Reply:
While implementing the second PR I've discovered a bug in this implementation for a specific case and I've adjusted the implementation a little bit

Reply:
Ok, but we do not take result of that method. What is the reason of checking if it exists, if we're not interested in result?

Reply:
This code has been changed, please see the updated version